File: PYTHON QUESTION/Sample_Checkout.py
all_items = ""
subtotal_all_items = 0


customer_name = input("What is the the customer's Name?: ")

while True: 

	item = input("What did the user buy?: ")

	quantity = int(input("How many pieces?: "))

	price = float(input("How many per unit?: "))
	
	# mulitiply quantity * per price to get the total price of all the item

	total = quantity * price
	
	subtotal_all_items = subtotal_all_items + total

	all_items = all_items + f"{item} x {quantity} = {total}\n"

	more = input("Add more item? (yes or no)")
	
	if more.lower() == "no":
		break

	




	cashier_name = input("What is your name cashier?")
	
	discount_percent = float(input("How much discount will he get?:"))
	

	discount = subtotal_all_items * discount_percent / 100;

	vat = subtotal_all_items * 7.5 / 100;  #(VAT stands for value added tax)

	total_bill = subtotal_all_items - discount + vat

	
	amount_paid = float(input("How much did the customer give you?"))
	
	balance = amount_paid - total_bill;



	print("\n================ INVOICE ================")
	print("Customer Name: ", customer_name)
	print("Cashier Name: ", cashier_name)
	print("Items Bought:", all_items)
	print("Sub Total All Items: ", subtotal_all_items)
	print("Discount: ", discount)
	print("VAT: ", vat)
	print("Total Bill: ", total_bill)
	print("Amount Paid: ", amount_paid)
	print("Balance: ", balance)
	print("=========================================")
	print("THANKS YOU FOR YOUR PATRONGE")





















	
	

File: JAVA QUESTION/SampleCheckout.java
import java.util.Scanner;
public class SampleCheckout {
	public static void main(String[] args) {
	Scanner input = new Scanner(System.in);



	System.out.println("What is the the customer's Name?: ");
	String customerName = input.nextLine();

	
	String allItems = "";
	double subtotalAllItems = 0;

	while(true) {

	System.out.println("What did the user buy?: ");
	String item = input.nextLine();

	System.out.println("How many pieces?: ");
	int quantity = input.nextInt();


	
	System.out.println("How many per unit?: ");
	double price = input.nextDouble();
	
	// mulitiply quantity * per price to get the total price of all the item

	double total = quantity * price;
	
	subtotalAllItems = subtotalAllItems + total;
	
	allItems = allItems + item + " x" + quantity + " =" + total + "\n";

	System.out.println("Add more item? (yes or no)");
	String more = input.nextLine();

	if (more.equalsIgnoreCase("no")) {
		break;

	}


}

	System.out.println("What is your name cashier?");
	String cashierName = input.nextLine();

	System.out.println("How much discount will he get?:");
	double discountPercent = input.nextDouble();

	double discount = subtotalAllItems * discountPercent / 100;

	double vat = subtotalAllItems * 7.5 / 100;  //(VAT stands for value added tax)

	double Totalbill = subtotalAllItems - discount + vat;

	
	System.out.println("How much did the customer give you?");
	double amountPaid = input.nextDouble();

	double balance = amountPaid - Totalbill;



	System.out.println("\n================ INVOICE ================\n");
	System.out.println("Customer Name: " + customerName);
	System.out.println("Cashier Name: " + cashierName);
	System.out.println("Items Bought:");
	System.out.println(allItems);
	System.out.println("Sub Total All Items: " + subtotalAllItems);
	System.out.println("Discount: " + discount);
	System.out.println("VAT: " + vat);
	System.out.println("Total Bill: " + Totalbill);
	System.out.println("Amount Paid: " + amountPaid);
	System.out.println("Balance: " + balance);
	System.out.println("\n=========================================\n");
	System.out.println("THANKS YOU FOR YOUR PATRONGE");


















































}
}